src/questionnaire.js

```diff
@@ -67,6 +67,24 @@ module.exports = [
       return answers.frontEnd;
     },
   },
+  {
+    name: "testCaseFramework",
+    type: "list",
+    message: "Select the Test Case Framework",
+    choices: [
+      { name: "MochaJS", value: "mochaJS" },
+      { name: "Jest", value: "jest" },
+      { name: "Jasmine", value: "jasmine" },
+      { name: "Karma", value: "karma" },
+      { name: "Puppeteer (Node Library)", value: "puppeteer" },
+      { name: "NightwatchJS", value: "nightwatchJS" },
+      { name: "Cypress", value: "cypress" },
+      { name: "None", value: false },
+    ],
+    when: (answers) => {
+      return answers.frontEnd;
```

Comment:
is the answer would save in this answers.frontEnd?
or what is variable that holds its answer

Reply:
when answers.frontEnd will have then it will show the options and **answers.testCaseFramework** holds value for it